website/docs/guides/migrating-to-azuread.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "azurerm"
+page_title: "Azure Active Directory: Migrating to the AzureAD Provider"
+sidebar_current: "docs-azurerm-migrating-to-azuread"
+description: |-
+  This page documents how to migrate from using the AzureAD resources within this repository to the resources in the new split-out repository.
+
+---
+
+# Azure Active Directory: Migrating to the AzureAD Provider
+
+In v1.21 of the AzureRM Provider the Azure Active Directory Data Sources and Resources have been split out into a new Provider specifically for Azure Active Directory.
+
+This guide covers how to migrate from using the following Data Sources and Resources in the AzureRM Provider to using them in the new AzureAD Provider:
+
+* Data Source: `azuread_application`
+* Data Source: `azuread_service_principal`
+* Resource: `azuread_application`
+* Resource: `azuread_service_principal`
+* Resource: `azuread_service_principal_password`
+
+## Updating the Provider block
+
+As the AzureAD and AzureRM Provider support the same authentication methods - it's possible to update the Provider block by setting the new Provider name and version, for example:
+
+```hcl
+provider "azurerm" {
+  version = "=1.21.0"
+}
+```
+
+can become:
+
+```hcl
+provider "azuread" {
+  version = "=0.1.0"
+}
+```
+
+## Updating the Terraform Configurations
+
+The Azure Active Directory Data Sources and Resources have been split out into the new Provider - which means the name of the Data Sources and Resources has changed slightly.
+
+The main difference in naming is that the `azurerm_` prefix has been removed from the names of the Data Sources and Resources - the following table explains the new name for each of the Azure Active Directory resources:
+
+
+| Type        | Old Name                                   | New Name                           |
+| ----------- | ------------------------------------------ | ---------------------------------- |
+| Data Source | azurerm_azuread_application                | azuread_application                |
+| Data Source | azurerm_azuread_service_principal          | azuread_service_principal          |
+| Resource    | azurerm_azuread_application                | azuread_application                |
+| Resource    | azurerm_azuread_service_principal          | azuread_service_principal          |
+| Resource    | azurerm_azuread_service_principal_password | azuread_service_principal_password |
+
+---
+
+Once the Provider blocks been updated, it should be possible to replace the `azurerm_` prefix in your Terraform Configuration from each of the AzureAD resources (and any interpolations) so that the new resources in the AzureAD Provider are used instead.
```

Comment:
Just noticed this. Should be "...Provider blocks have been updated,"

Reply:
👍 good spot